pkg/arvo/app/aqua.hoon

```diff
@@ -428,17 +480,47 @@
     this
   ==
 ::
+::  Make changes to azimuth state for the current fleet
+::
+++  poke-azimuth-action
+  |=  act=azimuth-action
+  ^-  (quip card:agent:gall _state)
+  =.  this  apex-aqua  =<  abet-aqua
+  ^+  this
+  ?-  -.act
+  ::
+      %init-azimuth
+    =.  azi.piers  *az-state
+    start-azimuth-timer
+  ::
+      %spawn
+    =.  state  (spawn who.act)
+    this
+  ::
+      %breach
+    ::  should we remove the pier from state here?
```

Comment:
No, because some of our tests are specifically to check that once a breach notification comes the defunct ship can't be reached.